Assets/Scripts/Camera/CameraController.cs

```diff
@@ -109,43 +103,42 @@ void HandleMovementInput()
             movementSpeed = normalSpeed;
         }
 
-        if(Input.GetKey(KeyCode.W) || Input.GetKey(KeyCode.UpArrow))
+        if(Input.GetAxis("Vertical") == 1)
```

Comment:
Axis shouldn't be pressed fully to make a movement. For example if it's only 0.5 then camera should have half of this speed.

Reply:
The same beneath.